testsuite/features/secondary/proxy_traditional_retail_pxeboot.feature

```diff
@@ -64,14 +59,14 @@ Feature: PXE boot a Retail terminal
     And I click on "Save Formula"
     Then I should see a "Formula saved" text
 
-@pxeboot_minion
   Scenario: Configure PXE part of DHCP on the branch server
     Given I am on the Systems overview page of this "proxy"
     When I follow "Formulas" in the content area
     And I follow first "Dhcpd" in the content area
     And I click on "Expand All Sections"
     And I enter the local IP address of "proxy" in next server field
     And I enter "boot/pxelinux.0" in filename field
+    And I enter "boot/pxelinux.0" in pxeboot filename field
```

Comment:
Interesting, both steps seems quite similar.

Reply:
Well, it also looks redundant in the product, if you ask me.

(in fact only the second is used ; but to be on the safe side i am putting both now)

(it was a test suite bug because it was Cobbler preparing the correct setting for Retail - it took me time to understand!)